core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -0,0 +1,128 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the body of the simple editor.
+ * editor.
+ */
+
+oppia.directive('simpleEditorBody', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    templateUrl: 'simpleEditor/body',
+    controller: [
+      '$scope', '$rootScope', '$timeout', 'SimpleEditorQuestionsDataService',
+      'explorationTitleService', 'explorationInitStateNameService',
+      'explorationStatesService',
+      function(
+          $scope, $rootScope, $timeout, SimpleEditorQuestionsDataService,
+          explorationTitleService, explorationInitStateNameService,
+          explorationStatesService) {
+        $scope.explorationTitleService = explorationTitleService;
+        $scope.questions = [];
+
+        $scope.$on('simpleEditorLoaded', function() {
```

Comment:
Thoughts on having ``'simpleEditorLoaded'`` be part a constant defined by the exploration editor controller?